complete/src/main/java/price/checker/domain/rpsy/StockPriceRepository.java

```diff
@@ -0,0 +1,47 @@
+package price.checker.domain.rpsy;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+import price.checker.domain.StockPrice;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Collection;
+import java.util.Date;
+import java.util.List;
+
+
+/**
+ * This is the interface that will be automatically implemented by Spring into a Bean
+ */
+@Repository
+//JpaRepository<{Entity Class}, {Data Type of Primary Key Field}>
+public interface StockPriceRepository extends JpaRepository<StockPrice,String>{
+
+    //symbol parameter will be assigned to the query parameter with index 1,
+    //if returned integer is 0, it means querying alpha vantage is required
+    @Query(value = "SELECT COUNT(*) " +

```

Comment:
If the purpose of this method is to check if a price exists for a given Symbol/date combination -- you should be able to override this method in CrudRepository (which JpaRepository extends) . using your StockPrice object as the ID. 

    Optional<T> findById(ID var1);